config/default/configmap/kfservice.yaml

```diff
@@ -17,7 +17,7 @@ data:
         },
         "pytorch": {
             "image": "gcr.io/kfserving/pytorchserver"
-        }
+        },
```

Comment:
sorry to introduce this issue, please also update https://github.com/kubeflow/kfserving/blob/master/install/v0.1.0/kfserving.yaml#L516

Reply:
Wow nice catch Hougang! 
@cliveseldon, happy to merge this in as soon as you get both of these.